Dockerfiles/agent/Dockerfile

```diff
@@ -40,8 +40,22 @@ RUN find / -maxdepth 1 -type f -name 'datadog-agent*_amd64.deb' ! -name "$DD_AGE
     usr/lib/x86_64-linux-gnu/libsystemd.so.0.21.0 \
     # self-test certificates that are detected (false positive) as private keys
     opt/datadog-agent/embedded/lib/python2.7/site-packages/Cryptodome/SelfTest \
- && if [ "$PYTHON_VERSION" = "2" ]; then rm -rf opt/datadog-agent/embedded/lib/python3.* || true; fi \
- && if [ "$PYTHON_VERSION" = "3" ]; then rm -rf opt/datadog-agent/embedded/lib/python2.* || true; fi \
+ && if [ "$PYTHON_VERSION" = "2" ]; then \
+        rm -rf \
+            opt/datadog-agent/embedded/bin/pip3* \
+            opt/datadog-agent/embedded/bin/pydoc* \
+            opt/datadog-agent/embedded/bin/python3* \
+            opt/datadog-agent/embedded/bin/pyvenv* \
+            opt/datadog-agent/embedded/include/python3* \
+            opt/datadog-agent/embedded/lib/*python3* || true ;\
```

Comment:
I realize there are additional py3-specific binaries that should be removed in the py2-only image:

* `/opt/datadog-agent/embedded/bin/2to3-3.7`
* `/opt/datadog-agent/embedded/bin/easy_install-3.7`
* `/opt/datadog-agent/embedded/bin/idle*`
* `/opt/datadog-agent/embedded/bin/pydoc*`

Could you add them all here?

In addition to this, I also realized that some binaries are present for both py2 and 3 (similar to `pip`) and should be symlinked accordingly. This should be fixed directly in the package though, so I'll open a separate PR to address this.